### DISCUSSION TITLE
suggestion to MantineSize

### DISCUSSION THREAD

**Discussion by Paraxdev** [mantinedev/mantine#95] [Feedback]
I Would love to be able to set a custom size value or be able to add with style something like:
```js
<Button color={"red"} variant={"outline"} size={"xl"} style={{scale: "15%"}}>
                        Go back.
</Button>
```
since something like this isn't working

**Comment by cstrat** [comment]
The `size` prop for a button is simply a mix of these styles:

https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Button/Button.styles.ts
``` 
const sizes = {  xs: {    fontSize: 10,    height: 22,    padding: [0, 11],  },
  sm: {    fontSize: 11,    height: 26,    padding: [0, 13],  },
  md: {    fontSize: 13,    height: 30,    padding: [0, 15],  },
  lg: {    fontSize: 14,    height: 36,    padding: [0, 18],  },
  xl: {    fontSize: 16,    height: 44,    padding: [0, 22],  },};
```

So you could just not include the `size` prop and just have the style prop to include those attributes with whatever values you want.

**Reply by Paraxdev** [reply]
ohh thats nice! thanks but i would love something like tailwind has, A config file where you can define custom values something like size xxl or more for example
